tajo-storage/tajo-storage-hbase/src/main/java/org/apache/tajo/storage/hbase/HBaseStorageManager.java

```diff
@@ -312,6 +312,17 @@ public static Configuration getHBaseConfiguration(Configuration conf, TableMeta
           HBaseStorageConstants.META_ZK_QUORUM_KEY + "' attribute.");
     }
 
+    String zkPort = hbaseConf.get(HConstants.ZOOKEEPER_CLIENT_PORT);
```

Comment:
I wonder why we don't use the default value.
Should users always specify the zookeeper client port?